### ISSUE TITLE
(ISO 8601-1/Amd 1) Refinement in ISO Editing DIS template for metadata

### ISSUE THREAD

**Issue by ronaldtse** [metanorma/metanorma-iso#683]
<img width="545" alt="Screenshot 2022-05-19 at 2 37 10 PM" src="https://user-images.githubusercontent.com/11865/169226386-8b8ba993-61e5-4016-8f46-97e8e60788c0.png">

1. Committee identification is not the working group, but the "approval group" which is typically one level up from the working group. We need to differentiate the "approval group" and the "working group".
2. Reference number is probably an open text that is not locked to "approval group" (i.e. "ISO/TC 154" in this case). Maybe we shouldn't generate the "ISO/TC 154" portion in the reference number.

_Originally posted by @ronaldtse in https://github.com/metanorma/metanorma-iso/issues/673#issuecomment-1131307782_

**Comment by opoudjis** [comment]
> Committee identification is not the working group, but the "approval group" which is typically one level up from the working group. We need to differentiate the "approval group" and the "working group".

Typically, or always? Do I need to add a new approval-group document attribute, or can I just use subcommittee + technical committee?

**Comment by opoudjis** [comment]
> Reference number is probably an open text that is not locked to "approval group" (i.e. "ISO/TC 154" in this case). Maybe we shouldn't generate the "ISO/TC 154" portion in the reference number.

What are you saying? That the  "ISO/TC 154" should have been included in the tc-document-number document attribute?

**Comment by ronaldtse** [comment]
> Typically, or always? Do I need to add a new approval-group document attribute, or can I just use subcommittee + technical committee?

Yes to add a new approval-group document attribute. It can differ in ways that we cannot imagine.

> That the "ISO/TC 154" should have been included in the tc-document-number document attribute?

I'm saying that "ISO/TC 154" should have been manual input. Again, the document can be circulated in some group with an N-number but the approval group can be another group, and the working group is yet another group.

**Comment by opoudjis** [comment]
If approval-group is not supplied, we will construct using technical committee ID.

**Comment by ronaldtse** [comment]
@opoudjis that's clever -- thank you!
